infra/bridge/src/controllers/promotions.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+const { getAsync } = require('../utils/redis')
+
+const Attestation = require('./../models/attestation')
+
+// TODO: Add request body validation
+router.post('/verify', async (req, res) => {
+  const { type, socialNetwork, identity } = req.body
+
+  const attestation = await Attestation.findOne({
+    ethAddress: identity,
+    method: socialNetwork
+  })
+
+  if (!attestation) {
+    return res.status(400).send({
+      success: false,
+      errors: [`Attestation missing`]
+    })
+  }
+
+  const redisKey = `${socialNetwork.toLowerCase()}/${type.toLowerCase()}/${
+    attestation.value
+  }`
+  let tries = 0
+  const interval = setInterval(async () => {
+    const event = await getAsync(redisKey)
+
+    if (event) {
+      clearInterval(interval)
+      // TODO: Process event and Push to growth events
+      return res.status(200).send({
+        success: true
+      })
+    } else {
+      tries++
+    }
+
+    if (tries >= 300) {
+      // Request will timeout after 1000ms * 300 === 300s or 5mins
+      clearInterval(interval)
+      return res.status(200).status({
+        success: false,
+        errors: [`Couldn't verify`]
+      })
+    }
+  }, 1000)
+})
+
+// // TODO: Add request body validation
```

Comment:
@franckc I added this code before checking out Activity API. Commented it out for now and the migration file is still there. Should I remove this or leave it so that we can probably use some of this later for social networks that don't support webhooks?

Reply:
I'd leave the code with a note explaining this is currently not being used.
If in a couple months we determined we won't need it, we can remove it.
As to the migration file, I'd remove it since otherwise it will get executed.
But perhaps have the table schema as a comment below ?